src/widgetastic_patternfly/__init__.py

```diff
@@ -1857,20 +1857,29 @@ class AboutModal(Widget):
 
     Provides a close method
     """
-    ROOT = ParametrizedLocator('//div[normalize-space(@id)={@id|quote} and '
-                               'contains(@class, "modal") and contains(@class, "fade")]')
+    # ROOT_LOC only used when id is not passed to constructor
+    ROOT_LOC = ('//div[contains(@class, "modal") and contains(@class, "fade") '
```

Comment:
Was this changed to `ROOT_LOC` to match the names of the other constants? Most of the other widgets still use `ROOT`

Reply:
using ROOT is to automagically resolve the `__locator__` method, since I don't intend for that to happen here I'm using ROOT_LOC since I directly implement `__locator__`.

Reply:
Makes sense, thanks!